packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -133,14 +133,15 @@ class ZapierBaseCommand extends Command {
    * @param {string} question the question to ask the user
    * @param {object} opts `inquierer.js` opts ([read more](https://github.com/SBoudrias/Inquirer.js/#question))
    */
-  async prompt(question, opts = {}) {
-    const { ans } = await inquirer.prompt({
-      type: 'string',
-      ...opts,
-      name: 'ans',
-      message: question
-    });
-    return ans;
+  prompt(question, opts = {}) {
+    return inquirer
+      .prompt({
+        type: 'string',
+        ...opts,
+        name: 'ans',
+        message: question
+      })
+      .then(({ ans }) => ans);
```

Comment:
Any reasons why changing it to promise-style vs. async/await?

Reply:
it _probably_ doesn't matter, but it brought it more in line with the other prompts.

In theory this has a performance benefit because it returns a promise to the calling function instead of waiting here and then returning. In practice, it probably doesn't matter. I'll revert.